users.view.lkml

```diff
@@ -61,6 +61,26 @@ view: users {
     sql: ${TABLE}.time_zone ;;
   }
 
+  dimension: bpo_site {
+    label: "BPO Site"
+    description: "Location of third-party service provider."
+    type: string
+    sql: ${TABLE}.user_fields__bpo_site ;;
+  }
+
+  dimension: team_lead {
+    description: "Name of the team lead responsible for the management of a particular agent."
+    type: string
+    sql: ${TABLE}.user_fields__team_lead ;;
+  }
+
+  dimension: bpo_start_date {
+    label: "BPO Start Date"
+    description: "Start date of an agent at a third-party service provider."
+    type: string
```

Comment:
Shouldn't this be a `dimension_group` of `time` and follow the timestamp conventions?

Reply:
@xiaolong I just updated this name of this field.  It's actually a string that is a concatenation of the cohorts starting week + the location of the cohort.